libs/java.json

```diff
@@ -40,10 +41,14 @@
   {"id":"java.dozer","imports":["org.dozer."],"name":"Dozer","repo":"DozerMapper/dozer","tags":["mapping"],"tech":["app-data"],"status":"model-ready"},
   {"id":"java.fastjson","imports":["com.alibaba.fastjson."],"name":"fastjson","repo":"alibaba/fastjson","tags":["json"],"tech":["app-data"],"status":"model-ready"},
   {"id":"java.gson","imports":["com.google.gson."],"name":"Gson","repo":"google/gson","tags":["json"],"tech":["app-data"],"status":"model-ready"},
+  {"id":"java.hibernate","imports":["org.hibernate"],"name":"Hibernate","repo":"hibernate/hibernate-orm","tags":["mapping"],"tech":["app-data"],"status":"awaiting-model"},
```

Comment:
tags == orm
tech == storage

It's what we've done for all ORM libraries before.